velox/vector/tests/VectorTest.cpp

```diff
@@ -396,9 +374,31 @@ class VectorTest : public testing::Test, public test::VectorTestBase {
     return odd;
   }
 
+  std::string printEncodings(const VectorPtr& vector) {
+    std::stringstream out;
+    out << vector->encoding();
+
+    VectorPtr inner = vector;
+    do {
+      switch (inner->encoding()) {
+        case VectorEncoding::Simple::DICTIONARY:
+        case VectorEncoding::Simple::SEQUENCE:
+        case VectorEncoding::Simple::CONSTANT:
+          inner = inner->valueVector();
+          if (inner == nullptr) {
+            return out.str();
+          }
+          break;
+        default:
+          return out.str();
```

Comment:
Curious, if its lazy would it be possible to get an idea of the inner encodings ?

Reply:
That cannot be done before lazy is loaded.